### PR TITLE
Update bootstrap.rst

### PR DESCRIPTION
**Pull request by Eoras**

### CODE REVIEW

File: frontend/encore/bootstrap.rst [hunk 1]
```diff
@@ -1,18 +1,24 @@
-Using Bootstrap CSS & JS
-========================
+Using Bootstrap 4 CSS & JS
```
Comment (javiereguiluz) [inline frontend/encore/bootstrap.rst]: Let's revert this change and remove the `4` version number. This article will be perpetually updated when new versions of Bootstrap are published, so there's no need to update the title.

File: frontend/encore/bootstrap.rst [hunk 2]
```diff
@@ -1,18 +1,24 @@
...
+============================
 
 Want to use Bootstrap (or something similar) in your project? No problem!
 First, install it. To be able to customize things further, we'll install
-``bootstrap-sass``:
+``bootstrap`` and popper.js:
```
Comment (javiereguiluz) [inline frontend/encore/bootstrap.rst]: Let's revert this change because the article is about Bootstrap, so we don't want to install other libraries. Thanks!

File: frontend/encore/bootstrap.rst [hunk 3]
```diff
@@ -1,18 +1,24 @@
...
 
 .. code-block:: terminal
 
-    $ yarn add bootstrap-sass --dev
+    $ yarn add bootstrap popper.js --dev
 
-Importing Bootstrap Sass
+or if you prefer to use npm:
```
Comment (javiereguiluz) [inline frontend/encore/bootstrap.rst]: We decided to use `yarn` in the Symfony Docs. This is of course a decision that can be debated and changed. But meanwhile we can't change individual articles. So, let's remove this change about "npm" and the other one later in this article.

File: frontend/encore/bootstrap.rst [hunk 4]
```diff
@@ -21,55 +27,31 @@ a ``global.scss`` file, import it from there:
...
-That's it! This imports the ``node_modules/bootstrap-sass/assets/stylesheets/_bootstrap.scss``
+That's it! This imports the ``node_modules/bootstrap/scss/bootstrap.scss``
 file into ``global.scss``. You can even customize the Bootstrap variables first!
 
 .. tip::
 
     If you don't need *all* of Bootstrap's features, you can include specific files
-    in the ``bootstrap`` directory instead - e.g. ``~bootstrap-sass/assets/stylesheets/bootstrap/alerts``.
-
-After including ``bootstrap-sass``, your Webpack builds might become slow. To fix
```
Comment (javiereguiluz) [inline frontend/encore/bootstrap.rst]: Just asking, why did you remove this explanation about `resolveUrlLoader`? Is it no longer needed in Bootstrap 4? Thanks.

**Review by javiereguiluz — CHANGES_REQUESTED** [review]
@Eoras thanks for this contribution! It contains some fixes that we definitely must do ... but also some changes that should be reverted for the reasons I gave in the inlined comments.

Thanks!

File: frontend/encore/bootstrap.rst (hunk 2 above)
Comment (Eoras) [reply frontend/encore/bootstrap.rst]: I delete popper.js but bootstrap 4 need it to work with tooltips. (Yarn and nmp say to add this dependancie when we install bootstrap so is ok)

File: frontend/encore/bootstrap.rst (hunk 4 above)
Comment (Eoras) [reply frontend/encore/bootstrap.rst]: I delete this because now on the scss file from bootstrap, we don't need to change the path from the webfont etc, so WebPack work realy fine now... i'm doing some test but for me isn't necessary anymore.

**Comment by javiereguiluz** [comment]
This is now merged! We merged in 3.4 and we'll merge in the other branches automatically. @Eoras thanks for your first Symfony Docs contribution!

File: frontend/encore/bootstrap.rst [hunk 5]
```diff
@@ -1,18 +1,18 @@
 Using Bootstrap CSS & JS
-========================
+============================
```
Comment (HeahDude) [inline frontend/encore/bootstrap.rst]: Should be reverted.

File: frontend/encore/bootstrap.rst [hunk 6]
```diff
@@ -1,18 +1,18 @@
...
 
 .. code-block:: terminal
 
-    $ yarn add bootstrap-sass --dev
-
-Importing Bootstrap Sass
+    $ yarn add bootstrap --dev
+    
+Importing Bootstrap 4
 ------------------------
```
Comment (HeahDude) [inline frontend/encore/bootstrap.rst]: Should be shortened.

File: frontend/encore/bootstrap.rst [hunk 7]
```diff
@@ -21,49 +21,21 @@ a ``global.scss`` file, import it from there:
...
-
-    + // set if you're also including font-awesome
-    + // $fa-font-path: "~font-awesome/fonts";
-
-    @import '~bootstrap-sass/assets/stylesheets/bootstrap';
+    in the ``bootstrap`` directory instead - e.g. ``~bootstrap/scss/alert``.
 
 Importing Bootstrap JavaScript
-------------------------------
+---------------------------------
```
Comment (HeahDude) [inline frontend/encore/bootstrap.rst]: Should be reverted.

File: frontend/encore/bootstrap.rst (hunk 5 above)
Comment (javiereguiluz) [reply frontend/encore/bootstrap.rst]: Done in c90f20a94e2e46d9ba2f9cedeafc2845627414ba. The other issues were fixed in 6b732a65b3e9c716d214dffa7461922e6c5ca0c6. Thanks for reporting.
